service/auth/authentication.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import User
+from rest_framework import authentication
+from rest_framework import exceptions
+from django.contrib.auth import login
```

Comment:
This file needs to import `ObjectDoesNotExist` and `from rest_framework import exceptions` and `from django.contrib.auth import login` can be removed since they are not being used.